simulators/ethereum/pyspec/main.go

```diff
@@ -40,9 +38,7 @@ func main() {
 	hivesim.MustRunSuite(hivesim.New(), suite)
 }
 
-// ------------------------------------------------------------------------- //
-// fixtureRunner() loads the pyspec test files and spawns the client tests.  //
-// ------------------------------------------------------------------------- //
+// fixtureRunner() loads the pyspec test files and spawns the client tests.
```

Comment:
Parentheses denoting a function are typically omitted in Go documentation comments.